pgcli_sublime.py

```diff
@@ -273,6 +268,26 @@ def run(self):
         self.window.active_view().set_scratch(True)
 
 
+def get_current_query(view):
+    text = get_entire_view_text(view)
+    cursor_pos = view.sel()[0].begin()
+
+    # Parse sql
+    stack = sqlparse.engine.FilterStack()
+    stack.split_statements = True
+    split_sql = [str(stmt) for stmt in stack.run(text)]
+    cum_len = 0
+
+    for query in split_sql:
```

Comment:
To avoid multiple redundant passes over the sql we could could rid of `split_sql = [str(stmt) for stmt in stack.run(text)]` and just do 
```
for query in stack.run(text):
    cum_len += len(str(query))
```